client/src/app/api/rest.ts

```diff
@@ -678,3 +680,10 @@ export const createTagCategory = (
 
 export const updateTagCategory = (obj: TagCategory): Promise<TagCategory> =>
   axios.put(`${TAG_CATEGORIES}/${obj.id}`, obj);
+
+// Facts
+
+export const getFacts = (id: number | string | undefined) =>
```

Comment:
The other query functions have an explicit return type Promise<?>.  Does it make sense to add one here?

Reply:
This one may fall under the same category as [this comment](https://github.com/konveyor/tackle2-ui/pull/1057/files#r1237604675). I am unsure how to type this unstructured fact that is just a json/yaml object that hasn't been parsed yet.